fatal error: libs/indiclient/baseclient.h: No such file or directory

### Expected Behaviour

Compiles fine.

### Actual Behaviour

Compilation failed. https://aur.archlinux.org/packages/stellarium#comment-1003064

```
In file included from /build/stellarium/src/build/plugins/TelescopeControl/src/INDI/TelescopeControl_INDI_autogen/EWIEGA46WW/moc_INDIConnection.cpp:9,
                 from /build/stellarium/src/build/plugins/TelescopeControl/src/INDI/TelescopeControl_INDI_autogen/mocs_compilation.cpp:2:
/build/stellarium/src/build/plugins/TelescopeControl/src/INDI/TelescopeControl_INDI_autogen/EWIEGA46WW/../../../../../../../stellarium-24.4/plugins/TelescopeControl/src/INDI/INDIConnection.hpp:23:10: fatal error: libs/indiclient/baseclient.h: No such file or directory
   23 | #include <libs/indiclient/baseclient.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

### Steps to reproduce

Install `libindi` https://archlinux.org/packages/extra/x86_64/libindi/files and enable `-DPREFER_SYSTEM_INDILIB=Yes`.

### System
* Stellarium version: 24.4
* Operating system: Arch Linux

--- a/plugins/TelescopeControl/src/INDI/INDIConnection.cpp
+++ b/plugins/TelescopeControl/src/INDI/INDIConnection.cpp
@@ -25,9 +25,9 @@
 #include <limits>
 #include <cmath>
 
-#include <libs/indiclient/baseclient.h>
-#include <libs/indidevice/basedevice.h>
-#include <libs/indibase/inditelescope.h>
+#include <libindi/baseclient.h>
+#include <libindi/basedevice.h>
+#include <libindi/inditelescope.h>
 
 const int INDIConnection::SLEW_STOP = INDI::Telescope::SLEW_GUIDE - 1;
 

--- a/plugins/TelescopeControl/src/INDI/INDIConnection.hpp
+++ b/plugins/TelescopeControl/src/INDI/INDIConnection.hpp
@@ -20,8 +20,8 @@
 #define INDICONNECTION_HPP
 
 #include <QObject>
-#include <libs/indiclient/baseclient.h>
-#include <libs/indidevice/basedevice.h>
+#include <libindi/baseclient.h>
+#include <libindi/basedevice.h>
 
 #include <mutex>
 #include <QStringList>

--- a/plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp
+++ b/plugins/TelescopeControl/src/INDI/TelescopeClientINDI.cpp
@@ -25,7 +25,7 @@
 
 #include "StelCore.hpp"
 #include "StelUtils.hpp"
-#include <libs/indibase/inditelescope.h>
+#include <libindi/inditelescope.h>
 #include "INDIControlWidget.hpp"
 
 TelescopeClientINDI::TelescopeClientINDI(const QString &name, const QString &params):

--- a/plugins/TelescopeControl/src/INDI/test/testINDIConnection.cpp
+++ b/plugins/TelescopeControl/src/INDI/test/testINDIConnection.cpp
@@ -23,7 +23,7 @@
 #include <string>
 
 #include "INDIConnection.hpp"
-#include <libs/indidevice/basedevice.h>
+#include <libindi/basedevice.h>
 
 void TestINDIConnection::deafultCoordinates()
 {

--- a/src/external/CMakeLists.txt
+++ b/src/external/CMakeLists.txt
@@ -189,6 +189,7 @@ IF(USE_PLUGIN_TELESCOPECONTROL AND ENABLE_INDI)
                 ${indiclient_SOURCE_DIR}/libindi
                 ${indiclient_SOURCE_DIR}/libs
                 PUBLIC
+                ${CMAKE_CURRENT_SOURCE_DIR}/fake-indi
                 ${CMAKE_CURRENT_BINARY_DIR}
                 ${indiclient_SOURCE_DIR}
                 ${indiclient_SOURCE_DIR}/libs/sockets

--- a/src/external/fake-indi/libindi/baseclient.h
+++ b/src/external/fake-indi/libindi/baseclient.h
@@ -1,0 +1,1 @@
+#include "libs/indiclient/baseclient.h"

--- a/src/external/fake-indi/libindi/basedevice.h
+++ b/src/external/fake-indi/libindi/basedevice.h
@@ -1,0 +1,1 @@
+#include "libs/indidevice/basedevice.h"

--- a/src/external/fake-indi/libindi/inditelescope.h
+++ b/src/external/fake-indi/libindi/inditelescope.h
@@ -1,0 +1,1 @@
+#include "libs/indibase/inditelescope.h"

